Update link and joint names

jimmy-mcelwain opened the pull request:
I am just looking for initial feedback. Marking this as draft. 

The `sda10f` package is incomplete. I will fix it before finalizing this PR. 

The main things I would like to get feedback on:
- I changed the names of the `stl` and `dae` files and modified the link names within the `dae` files. I'm fine with reverting either of these changes. 
- I changed the names of balancers named `link_1_s_2`, `link_2_l_2` to `balancer_1`, `balancer_2`
- For SDA packages, I'll probably go with [this](https://gavanderhoorn.github.io/rep/rep-0199.html#multi-group-symmetric), but just wanted to ask if we should instead go with naming them `group_1`, `group_2`, `group_3` instead of `right` and `left`. This makes it a bit less understandable, but it would match the default `/joint_states` names. 

Also this is not very important, but I think that naming the joint `link_x-flange` makes more sense than `joint_x-flange`. It seems like most other manufacturers do `joint_x-flange`, though so maybe it's best to just leave it.

jimmy-mcelwain committed:
changed link mesh file names for GP series arms

jimmy-mcelwain committed:
Rename mesh files for remaining single arm robots and positioners

jimmy-mcelwain committed:
Change names of motoman_es_support link meshes

jimmy-mcelwain committed:
Change references to all renamed mesh files

jimmy-mcelwain committed:
Update link and joint names in `xacro` files

jimmy-mcelwain committed:
Update joint names in config files

jimmy-mcelwain committed:
Change joint names in package manifests

jimmy-mcelwain committed:
Change names in `.dae` files

jimmy-mcelwain commented:
@gavanderhoorn @ted-miller ping -- there will be several big PRs that touch all packages soon after this one gets merged. I am talking about #3, #12, potentially #16, potentially #26, #29. For that reason, I don't think it is important to be thorough with this PR, but I would like to know your opinions on my notes in my earlier comment. Thanks!

ted-miller commented:
> I changed the names of the stl and dae files and modified the link names within the dae files. I'm fine with reverting either of these changes.

Yes, I agree with removing the "slurbt" from the link names.

> I changed the names of balancers named link_1_s_2, link_2_l_2 to balancer_1, balancer_2

`balancer` is much more clear than `xxx_2`

> For SDA packages, I'll probably go with [this](https://gavanderhoorn.github.io/rep/rep-0199.html#multi-group-symmetric), but just wanted to ask if we should instead go with naming them group_1, group_2, group_3 instead of right and left. This makes it a bit less understandable, but it would match the default /joint_states names.

I'm in favor of matching the names of `joint_states`.

> Also this is not very important, but I think that naming the joint link_x-flange makes more sense than joint_x-flange. It seems like most other manufacturers do joint_x-flange, though so maybe it's best to just leave it.

I have no opinion on this. Whatever you want is fine.

jimmy-mcelwain commented:
@gavanderhoorn thoughts?

jimmy-mcelwain committed:
change name of sda meshes

jimmy-mcelwain committed:
change references to sda meshes

jimmy-mcelwain committed:
Change SDA link, joint names, and add new prefixes

jimmy-mcelwain committed:
Add base frames

jimmy-mcelwain committed:
Remove outdated xacro

jimmy-mcelwain commented:
I cleaned up the history, fixed the sda package, and got it ready to merge. 

I made one or two non-name related changes to the SDA package. 

The each SDA `xacro` has 2 default invocations of the `macro_xacro`. One that should match up with MotoROS2 defaults `group_x/joint_x` and one that more closely aligns with the REP I linked above (it still doesn't perfectly align, though). 

Everything renders and looks right. I have a tool that makes it easy to check sequentially in maybe 3 minutes that I can share if anybody wants it

jimmy-mcelwain commented:
I will resolve these conflicts soon, but then I would like to merge this.

jimmy-mcelwain commented:
Rebased. It would be good to get this merged @ted-miller @gavanderhoorn.

ted-miller reviewed (approved):
Looks good. Much more consistent.
